test/api-views/dashboard.test.js

```diff
@@ -0,0 +1,24 @@
+const dbHandler = require("../db-handler");
+const supertest = require("supertest");
+const { setupTestApp } = require("../../app-test");
+
+let request;
+
+beforeAll(async () => {
+  await dbHandler.connect();
+  const server = await setupTestApp();
+
+  request = supertest(server);
+});
```

Comment:
These lines are all we need to setup an API test now! Lot better if you ask me.

Reply:
Yeah this is well easier, might even get me doing it more xD